pkg/controller/controller.go

```diff
@@ -87,16 +88,16 @@ func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
 	}()
 	glog.Infoln("Starting CDI controller loop")
 	if threadiness < 1 {
-		return fmt.Errorf("controller.Run: expected >0 threads, got %d", threadiness)
+		return Errf("expected >0 threads, got %d", threadiness)
```

Comment:
see above comment re. when to call `Errf`. Maybe we don't ever want/need _main_ in the log so then the approach would be for the highest level func below main to call Errf?